notebooks/community/ml_ops/stage7/model_monitoring_setup.ipynb

```diff
@@ -502,6 +502,84 @@
         "- For the CSV data source, grant both service accounts the \"Storage Object Viewer\" role."
```

Comment:
<p>Typo: The predefined <code>input schema</code> must be <strong><em>loaded</em></strong> to a Cloud Storage location. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
done